etc/kayobe/kolla/config/prometheus/prometheus-alertmanager.yml.example

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
It would be helpful to provide a few comments about the particular config used here - splitting critical alerts into a separate channel etc.

Reply:
Done